editor/src/core/es-modules/package-manager/package-manager.ts

```diff
@@ -91,11 +105,11 @@ export function getRequireFn(
             // we should extend the module objects so it not only contains the exports,
             // to have feature parity with the popular bundlers (Parcel / webpack)
             // MUTATION
-            resolvedFile.evalResultCache = { module: partialModule }
+            //  resolvedFile.evalResultCache = { module: partialModule }
```

Comment:
Does this (and the commented line below) need to go in the bin or was this a temporary commenting?

Reply:
Yes, good spot.

Reply:
Fixed in 1e264e2b.